File: mi-carpeta/deber_5_if2.ts
// Mayor entre dos numeros
console.log("\nMayor entre dos numeros\n");
const nu1String = prompt("Ingrese su primera numero: ");
const nu1 = Number(nu1String);
console.log("\nSaber si tienes que ir a vielar al final del semestre\n");
const nu2String = prompt("Ingrese su segundo numero: ");
const nu2 = Number(nu2String);
if(nu1>nu2){
    console.log("El numero mayor es el primero: ");
}
else if(nu2>nu1){
    console.log("El mumero mayor es el segundo: "+nu2);
}

File: mi-carpeta/deber_5_if2.go
// Mayor entre dos numeros
package main

import (
	"fmt"
)

func main() {
	fmt.Println("El mayor entre dos dumeros")
	var num1, num2 float64
	fmt.Println("Ingrese el primer numero: ")
	fmt.Scanln(&num1)
	fmt.Println("Ingrese el segundo numero: ")
	fmt.Scanln(&num2)
	if num1 > num2 {
		fmt.Println("El primero numero es el mayor: ", num1)
	} else {
		fmt.Println("El segundo numero es el mayor: ", num2)
	}
}
